packages/teams-js/src/public/visualMedia.ts

```diff
@@ -0,0 +1,274 @@
+     */
+    cameraSwitcher?: boolean;
+    /**
+     * max limit of media allowed to be selected in one go, current max limit is 10 set by lens-sdk.
+     */
+    visualMediaCount: number;
+  }
+  /**
+   *  All properties in ImageProps are optional and have default values in the platform
+   *  Additional properties for image in mobile devices
+   * @beta
+   */
+  export interface ImageProperties extends VisualMediaProps {
+    /**
+     * Optional; indicate if inking on the selected Image is allowed or not
+     * Default value is true
+     */
+    ink?: boolean;
+
+    /**
+     * Optional; indicate if putting text stickers on the selected Image is allowed or not
+     * Default value is true
+     */
+    textSticker?: boolean;
+
+    /**
+     * Optional; indicate if image filtering mode is enabled on the selected image
+     * Default value is false
+     */
+    enableFilter?: boolean;
+
+    /**
+     * Optional; Lets the developer specify the image output formats, more than one can be specified.
+     * Default value is Image.
+     */
+    imageOutputFormats?: ImageOutputFormats[];
+  }
+
+  /**
+   * The modes in which camera can be launched
+   * @beta
+   */
+  export enum CameraStartMode {
+    /** Photo mode. */
+    Photo = 1,
+    /** Document mode. */
+    Document = 2,
+    /** Whiteboard mode. */
+    Whiteboard = 3,
+    /** Business card mode. */
+    BusinessCard = 4,
+  }
+  /**
+   * Enum for file formats supported
+   * @beta
+   */
+  export enum FileFormat {
+    /** Base64 encoding */
+    Base64 = 'base64',
+    /** File id */
+    ID = 'id',
+  }
+  /**
+   * Specifies the image source
+   * @beta
+   */
+  export enum Source {
+    /** visual media source is camera. */
+    Camera = 1,
+    /** visual media source is gallery. */
+    Gallery = 2,
+  }
+  /**
+   * Specifies the image output formats.
+   * @beta
+   */
+  export enum ImageOutputFormats {
+    /** Outputs image.  */
+    IMAGE = 1,
+    /** Outputs pdf. */
+    PDF = 2,
+  }
+
+  /**
+   * VisualMediaFile object that can be used to represent image or video
+   *
+   * @beta
+   */
+  export class VisualMediaFile {
+    /**
+     * Content of the file. When format is Base64, this is the base64 content
+     * When format is ID, this is id mapping to the URI
+     * When format is base64 and app needs to use this directly in HTML tags, it should convert this to dataUrl.
+     */
+    public content: string;
+
+    /**
+     * Format of the content
+     */
+    public format: FileFormat;
+
+    /**
+     * Size of the file in KB
+     */
+    public size: number;
+
+    /**
+     * MIME type. This can be used for constructing a dataUrl, if needed.
+     */
+    public mimeType: string;
+
+    /**
+     * Optional: Name of the file
+     */
+    public name?: string;
+    /**
+     * A preview of the file which is a lightweight representation.
+     * In case of images this will be a thumbnail/compressed image in base64 encoding.
```

Comment:
>In case of images this will be a thumbnail/compressed image in base64 encoding. [](http://example.com/codeflow?start=7&length=79)

And when not images?

Reply:
Should be a preview of the video file. I will check more about the video functionality and come back  form this comment.